piety-promoted-v1/modernized/01-to-the-reader.adoc

```diff
@@ -176,33 +176,40 @@ having already "`their inheritance sealed unto them by the Spirit of God.`"
 
 Some said as follows:
```

Comment:
In order to remove the indent from this short phrase "Some said as follows:", lets mark it with an [.offset].  This will both remove the indent, and space it down a bit from the paragraph above. 👍🏻